### PR TITLE
Adds some extra categories

### PR DESCRIPTION
**Pull request by cluening**
Individual words to hunt for are denoted with quotes around them.

### CODE REVIEW

File: categories.json [hunk 1]
```diff
@@ -21,5 +21,17 @@
     "OH without irony: \"Microsoft has really gotten better at x\"",
     "Behaviour inspires new sub-clause in code-of-conduct",
     "Live demo worked first try",
-    "Tutorial 3GB VM install / malware orgy"
+    "Tutorial 3GB VM install / malware orgy",
+    "Devop (singular)",
+    "Google Glass",
```
Comment (stpierre) [inline categories.json]: Already covered by line 7, "Non-wrist wearable". Remove.

File: categories.json [hunk 2]
```diff
@@ -21,5 +21,17 @@
...
+    "Tinfoil hats",
+    "Uncomfortable conversation",
+    "3/4 through talk, 1/4 through slides",
+    "\"Devops\"",
+    "\"Noops\"",
+    "\"as a service\"",
+    "\"Cattle, not pets\"",
+    "\"10x\"",
```
Comment (stpierre) [inline categories.json:32]: Let's add "20x" too, either as a separate category or as an option here (e.g., `"\"10x\"/\"20x\""`).

File: categories.json [hunk 3]
```diff
@@ -21,5 +21,17 @@
     "OH without irony: \"Microsoft has really gotten better at x\"",
     "Behaviour inspires new sub-clause in code-of-conduct",
     "Live demo worked first try",
-    "Tutorial 3GB VM install / malware orgy"
+    "Tutorial 3GB VM install / malware orgy",
+    "Devop (singular)",
+    "Google Glass",
+    "Tinfoil hats",
```
Comment (stpierre) [inline categories.json:26]: Literal or figurative?

File: categories.json [hunk 4]
```diff
@@ -21,5 +21,17 @@
...
+    "Uncomfortable conversation",
+    "3/4 through talk, 1/4 through slides",
+    "\"Devops\"",
```
Comment (stpierre) [inline categories.json]: This appears in the program, so it's basically a free spot. Let's remove it. (Somewhat incredibly, neither "Noops" nor "as a Service" appear in the program AFAICT.)

File: categories.json [hunk 5]
```diff
@@ -21,5 +21,17 @@
...
+    "\"Noops\"",
+    "\"as a service\"",
```
Comment (stpierre) [inline categories.json]: Let's make this `"\"as-a-Service\" abuse\"`. We're guaranteed to hear IaaS and SaaS and so on, but I'm sure some marketroid has a truly egregious *-as-a-Service for us all to laugh at.

File: categories.json [hunk 6]
```diff
@@ -21,5 +21,17 @@
...
+    "\"Cattle, not pets\"",
+    "\"10x\"",
+    "\"Container\"",
```
Comment (stpierre) [inline categories.json]: "Container" is in the program, please remove.

**Review by stpierre — CHANGES_REQUESTED** [review]

File: categories.json (hunk 3 above)
Comment (cluening) [reply categories.json:26]: Literal, of course.

File: categories.json (hunk 2 above)
Comment (cluening) [reply categories.json:32]: I like adding it separately as something more difficult to get.

**Pull request merged**